src/python/pants/util/BUILD

```diff
@@ -142,6 +144,7 @@ python_library(
 python_library(
   name = 'tarutil',
   sources = ['tarutil.py'],
+  tags = {'type_checked'},
```

Comment:
This one is way too hard to actually type. We keep the file as is so that it uses type hints from the stdlib.